src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -676,7 +663,7 @@ suite('DataScience - VSCode Notebook - (Execution) (slow)', function () {
         }
     });
 
-    test('Handling of carriage returns', async () => {
+    test.only('Handling of carriage returns', async () => {
```

Comment:
What does this do?

Reply:
oops, shoot, good pick, that only tests that one test locally

Reply:
Even on CI, it would have run that one test